src/EpiceaBrowsers/EpDashboardPresenter.class.st

```diff
@@ -17,6 +17,20 @@ Class {
 	#category : #'EpiceaBrowsers-UI-Dashboard'
 }
 
+{ #category : #specs }
+EpDashboardPresenter class >> defaultSpec [
+	^ SpBoxLayout newVertical
+		add: #currentSessionInfoPresenter;
+		add:
```

Comment:
Hi, 

Sorry for arriving late (I'm having a huge queue of issues to review and slowly walking them). 
The usage of `#add:withConstraints:` to force a height is bad. 
Instead,  we use `#add:expand:fill:padding:`. In this example the code will be: 

```Smalltalk
EpDashboardPresenter class >> defaultSpec [
 	^ SpBoxLayout newVertical
 		add: #currentSessionInfoPresenter;
 		add: (SpBoxLayout newHorizontal
 				add: #enableMonitorButtonPresenter;
 				add: #createLogButtonPresenter;
 				add: #openSettingsButtonPresenter;
 				yourself)
			expand: false
			fill: false
			padding: 0;
		yourself
```
Please, can you fix this?

Reply:
Done in https://github.com/pharo-project/pharo/pull/4474